src/libs/pluginprocess/pluginprocess.c

```diff
@@ -0,0 +1,379 @@
+/**
+ * @file
+ *
+ * @brief Source for the pluginprocess library
+ *
+ * Executes plugins in a separate process via fork and uses a simple
+ * communication protocol based on the dump plugin via named pipes.
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include "kdbpluginprocess.h"
+#include <kdbinvoke.h>
+// To access the plugin function pointers
```

Comment:
Which header? Write it next to the include.

And why is this needed? Is kdbinvoke.h missing something?